pyproject.toml

```diff
@@ -1,12 +1,81 @@
 [build-system]
-build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools >= 46.6.0",
+    "setuptools",
+    "lsst-versions >= 1.3.0",
     "wheel",
     "pybind11 >= 2.5.0",
     "numpy >= 1.18",
-    "lsst-versions",
 ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-sphgeom"
+description = "A spherical geometry library."
+license = {text = "GPLv3+ License"}
+readme = "README.md"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering :: Astronomy",
+]
+keywords = ["lsst"]
+dependencies = [
+   "numpy >=1.18",
+   "healpy"
+]
+dynamic = ["version"]
+
+[project.urls]
+"Homepage" = "https://github.com/lsst/sphgeom"
+
+[project.optional-dependencies]
+test = [
+    "pytest >= 3.2",
+    "flake8 >= 3.7.5",
+    "pytest-flake8 >= 1.0.4",
+]
+yaml = ["pyyaml >= 5.1"]
+
+[tool.setuptools.packages.find]
+where = ["python"]
+
+[tool.setuptools]
+zip-safe = false
+license-files = ["LICENSE"]
+
+[tool.setuptools.dynamic]
+version = { attr = "lsst_versions.get_lsst_version" }
+
+[tool.black]
+line-length = 110
+target-version = ["py38"]
+
+[tool.isort]
```

Comment:
Oh, we hadn't run isort or black before. That explains the failures. Please fix the formatting problems.